<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPackage</class>
 <widget class="QDialog" name="AddPackage">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddPackage</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtPackageName"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="PackageName">
      <property name="text">
       <string>Pacakge Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Description">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="txtDescription"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Amount">
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtAmount"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
